ui/chains/ethereum/run-ethereum-test-chain.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+MNEMONIC=$(grep MNEMONIC .env | cut -d '=' -f 2-)
+BLOCKSPEED=0
+ganache-cli -m "$MNEMONIC"  -b "$BLOCKSPEED"
```

Comment:
Might be a good idea to run `yarn ganache-cli`. That was we don't need to necessarily install it globally.

Reply:
https://github.com/Sifchain/sifnode/blob/7770158cfef548ba553e1dc04d7ba3ec8e557b86/ui/chains/ethereum/run-ethereum-test-chain.sh#L5